lib/metadata/linux/LinuxUtils.rb

```diff
@@ -110,5 +115,29 @@ def self.parse_chkconfig_list(lines)
 
       return ret
     end
+
+    def self.parse_systemctl_list(lines)
+      ret = []
+      return ret if lines.nil? || lines.empty?
+
+      lines.each_line do |line|
+        line = line.chomp
+        parts = line.split(' ')
+        next unless /^.*?\.service$/ =~ parts[0]
+
+        name, _ = parts[0].split('.')
+
+        # TODO(lsmola) investigate adding systemd targets, which are used instead of runlevels. Drawback, it's not
+        # returned by any command, so we would have to parse the dir structure of /etc/systemd/system/
+        # There is already MiqLinux::Systemd.new(@systemFs) called from class MIQExtract, we should leverage that
```

Comment:
regarding this TODO, what do you use MIQExtract for and why is it separated from ssh fleecing? Is there some plans for merging them? Or they are supposed to be for different use cases?